CONTRIBUTING.md

```diff
@@ -56,16 +56,19 @@ We use web components from https://github.com/Qiskit/web-components to include c
 To update the top nav bar web component:
 
 1. In https://github.com/Qiskit/web-components, run `npm install` then `npm run build`.
-2. There should be a file created at the root of the web components repository called `experimental-bundled-ui-shell.js`. Copy its contents into the file `js/web-components/top-nav-bar.js` in this theme repository. 
-3. Build the example docs with `tox -e py` to ensure everything works.
+2. There should be a file created at the root of the web components repository called `experimental-bundled-ui-shell.js`. Copy its contents into these files in this theme repository:
+   1. `qiskit_sphinx_theme/pytorch_base/static/js/web-components/top-nav-bar.js`
```

Comment:
Like, rename the folder from `pytorch_base` to `legacy`? I used to call it `legacy_pytorch_base` but Abby asked me to get rid of the `legacy` part because it was too verbose. 

This folder name doesn't impact end users at all - only developers of this theme.

Reply:
Let's stick with the current naming. This folder name is internal.